packages/renderer/src/lib/task-manager/TaskManagerItem.svelte

```diff
@@ -111,6 +115,21 @@ function gotoTask(taskUI: StatefulTaskUI) {
       </div>
     {/if}
 
+    {#if isStatefulTask(taskUI) && taskUI.status !== 'failure'}
+      <div class="flex flex-row w-full">
+        <div class="flex flex-1 flex-col w-full items-end text-purple-500 text-xs">
+          {#if taskUI.openFolder}
+            <button
+              class="text-purple-500 cursor-pointer"
+              on:click="{() => {
+                if (isStatefulTask(taskUI)) openFolder(taskUI);
+              }}"
+              aria-label="open folder">Open folder ></button>
```

Comment:
```suggestion
              aria-label="open folder">Open folder ></button>
```

is that the `>` after Open Folder is wanted ?

Reply:
I did that on purpose to follow the open task but it can be removed.
![open_folder](https://github.com/containers/podman-desktop/assets/49404737/88aafce0-af0b-45f4-a5ea-b424a055c573)

Reply:
maybe use `&gt;` so we know it's an html entity ?

Reply:
done